src/pages/Contests/Contests.module.scss

```diff
@@ -19,3 +19,7 @@
     width: 40%;
   }
 }
+
+.finishedContest {
+  opacity: 0.6;
```

Comment:
Would it be nice to have it come to full opacity when hovered?

Reply:
Ah yes, I like that.